### PR TITLE
Share link

### PR DESCRIPTION
**Pull request by Ezeko**

### CODE REVIEW

File: share-link.html [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+                </div>
+         
+       
+    </div>
+    <script>
+        let facebook_link = document.querySelector('.fa-facebook')
+        facebook_link.href = `https://www.facebook.com/sharer/sharer.php?u=${window.location.href}`
+
+        let twitter_link = document.querySelector('.fa-twitter')
+        let cert_title = document.querySelector('#cert-title').innerHTML
```
Comment (shonubijerry) [inline share-link.html]: There is no element with an id="cert-title". Can you explain the workflow of this twitter share link? I understand

**Review by shonubijerry — CHANGES_REQUESTED** [review]
Can you move this file to `./src/views/template` folder? From there the other frontend guys can integrate it with the codebase.

**Comment by Ezeko** [comment]
Okay

**Review by shonubijerry — APPROVED** [review]
You didn't make the changes as requested, but I will merge and fix it.

**Pull request merged**